docs/sources/datasources/pyroscope/configure-pyroscope-data-source.md

```diff
@@ -33,98 +33,88 @@ refs:
       destination: /docs/grafana/<GRAFANA_VERSION>/datasources/tempo/configure-tempo-data-source/
     - pattern: /docs/grafana-cloud/
       destination: docs/grafana-cloud/connect-externally-hosted/data-sources/tempo/configure-tempo-data-source/
+  provisioning-data-sources:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/provisioning/#data-sources
+    - pattern: /docs/grafana-cloud/provision
+      destination: /docs/grafana/<GRAFANA_VERSION>/administration/provisioning/#data-sources
 ---
 
 # Configure the Grafana Pyroscope data source
 
+The Pyroscope data source sets how Grafana connects to your Pyroscope database.
+
+You can configure the data source using either the data source interface within Grafana or using a configuration file.
+This page explains how to set up and enable the data source capabilities using Grafana.
+
+If you make any changes, select **Save & test** to preserve those changes.
+
+If you are using your own installation of Grafana, you can provision the Pyroscope data source using a YAML configuration file.
+For more information about provisioning and available configuration options, refer to [Provisioning Grafana](ref:provisioning-data-sources).
+
+## Before you begin
+
+To configure a Pyroscope data source, you need administrator rights to your Grafana instance and a Pyroscope instance configured to send data to Grafana.
+
+If you are provisioning a Pyroscope data source, then you also need administrative rights on the server hosting your Grafana instance.
+
+## Set up the data source
+
 To configure basic settings for the data source, complete the following steps:
 
 1. Click **Connections** in the left-side menu.
 1. Under Your connections, click **Data sources**.
 1. Enter `Grafana Pyroscope` in the search bar.
 1. Select **Add new data source**.
 1. Click **Grafana Pyroscope** to display the **Settings** tab of the data source.
-1. Set the data source's basic configuration options.
+1. On the **Settings** tab, set the data source's basic configuration options. At a minimum, you need to complete the **Name**, **Connection**, and **Authentication** sections. The other sections provide optional capabilities.
```

Comment:
Just a thought, for this step, I would say something like: "Refer to the following table to complete configuration options on the Settings tab.

Field name                Description
Name                        Add the content you have in the H2 below
Connection              Add the content you have in the H2 below
etc.

Just a way to remove some H2 bulk from the topic.

Reply:
Sure! I can do that here an din the Pyroscope content as well.

Reply:
Done.